client/my-sites/comments/controller.js

```diff
@@ -13,32 +13,33 @@ import route from 'lib/route';
 import CommentsManagement from './main';
 import controller from 'my-sites/controller';
 
-export const isStatusValid = status => {
-	const validStatuses = [ 'pending', 'approved', 'spam', 'trash', 'all' ];
-	return includes( validStatuses, status );
+const VALID_STATUSES = [ 'pending', 'approved', 'spam', 'trash', 'all' ];
+
+export const isValidStatus = status => {
+	return includes( VALID_STATUSES, status );
 };
```

Comment:
fun fact! we're only interested in returning the evaluation of the expression here, so we don't actually need the noise from the curlies and the `return`

```js
export const isValidStatus = status => includes( VALID_STATUSES, status );
```